WeakAuras/Prototypes.lua

```diff
@@ -2618,7 +2624,7 @@ WeakAuras.event_prototypes = {
             end
 
           elseif (event == "DBM_TimerStop") then
-            if (states[id]) then
+            if (states[id] and states[id].extend == 0) then
```

Comment:
That's incorrect. the TimerStop event comes in two cases. One if the timer is finished, and secondly if the timer is obsolete, e.g. phase transitions.

Reply:
Indeed, "manually" hiding the bar on TimerStop event should be done if
- we didn't extend timer `states[id].extend == 0` (autoHide = false)
- remainingTime on dbm's timer is not 0 (obselete timer case)

In practice remainingTime is never exactly 0 when we get the TimerStop event
![results](https://i.imgur.com/Admpiv9.png)

So instead of comparing the bw/dbm timer's remainingTime with 0, i have chosen in the next commit to account for a 0.2s latency.
I don't like using this made up max latency but i don't see a better solution here.